files/Aboot/boot0.j2

```diff
@@ -608,15 +643,20 @@ run_kexec() {
 
 secureboot_install() {
     if [ -e "$image_path" ]; then
-        warn "Image folder $image_path already exist, wiping..."
-        rm -rf "$image_path"
+        warn "Image folder $image_path already exist (likely regular install)"
     fi
 
+    mkdir -p "$image_path"
```

Comment:
>mkdir -p "$image_path" [](start = 4, length = 22)

Wondering what is the different from original code?

Reply:
Previously there was a `mkdir $image_path` inside `extract_secureboot_image` which would fail if a regular image was already installed.
Instead I decided to move it away from the function and into `secureboot_install` to mimic the behavior of `regular_install`.